src/components/dialogs/BiometryDeniedDialog.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { useWebviewIntent } from 'cozy-intent'
+
+import { Button } from 'cozy-ui/transpiled/react/Button'
+import { Dialog } from 'cozy-ui/transpiled/react/CozyDialogs'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+
+interface BiometryDeniedDialogProps {
+  setModalVisible: React.Dispatch<React.SetStateAction<boolean>>
+}
+
+export const BiometryDeniedDialog = ({
+  setModalVisible
+}: BiometryDeniedDialogProps): JSX.Element => {
+  const onCancel = (): void => setModalVisible(false)
+  const { t } = useI18n()
+  const webviewIntent = useWebviewIntent()
+
+  const tryOpenSettings = (): void => {
+    if (!webviewIntent) return
+
+    void webviewIntent.call('openAppOSSettings').then(() => {
+      return setModalVisible(false)
+    })
+  }
+
+  return (
+    <Dialog
+      open
+      onClose={onCancel}
+      size="small"
+      title={t('BiometryDeniedDialog.title')}
+      actions={
+        <>
+          <Button
+            onClick={onCancel}
+            theme="secondary"
+            label={t('BiometryDeniedDialog.cancel')}
+          />
+          <Button
+            onClick={(): void => tryOpenSettings()}
+            label={t('BiometryDeniedDialog.open_settings')}
+          />
+        </>
+      }
+      content={
+        <>
+          <Typography className="u-mb-1" variant="body1">
+            {t('BiometryDeniedDialog.content')}
+          </Typography>
+        </>
```

Comment:
Nit: I think it’s the first time I’be to read actions before the content. You shifted my mental model here 😂

Reply:
This is because you didn't reviewed https://github.com/cozy/cozy-settings/pull/495 :trollface: 

I did a copy-paste of the PinCodeDialog component as a basis and I didn't even notice the ordering. I reversed the order